app/models/voucher.rb

```diff
@@ -167,6 +167,7 @@ def inverted_rows(options = {})
     rows.reject!(&:canceled?) unless options[:canceled]
     rows.map do |old|
       vr = old.clone
```

Comment:
> That means that modifying attributes of the clone will modify the original, since they will both point to the same attributes hash. If you need a copy of your attributes hash, please use the #dup method.

http://apidock.com/rails/ActiveRecord/Core/clone